examples/snippets/utils/client/manual_auth.py

```diff
@@ -0,0 +1,26 @@
+"""
+this snippet shows how a tamr client object can be created
+with manual passing of credentials during runtime.
+"""
+from tamr_toolbox import utils as tbu
+from tamr_toolbox.utils.client import Client
+from getpass import getpass
+
+
+def manual_auth(config: dict) -> Client:
```

Comment:
As a snippet, this can be a lot simpler, and doesn't need an encapsulating function.  I would just keep the imports, and then write the few necessary lines to get the input and create the `Client` object.

Reply:
Implemented the requested change.